processor/cumulativetodeltaprocessor/README.md

```diff
@@ -9,18 +9,27 @@ The cumulative to delta processor (`cumulativetodeltaprocessor`) converts cumula
 
 ## Configuration
 
-Configuration is specified through a list of metrics. The processor uses metric names to identify a set of cumulative sum metrics and converts them to cumulative delta.
+The default configuration is to convert all monotonic sum metrics from aggregation temporality cumulative to aggregation temporality delta.
+
+The following settings can be optionally configured:
+
+- `metrics`: The processor uses metric names to identify a set of cumulative sum metrics and converts them to cumulative delta. Defaults to converting all metric names.
```

Comment:
Similar question here. Do you have a use case where all metrics need to be translated? I think it would be the best if we will add regex support in future instead.

Reply:
New Relic *requires* all metrics to be translated from cumulative -> delta temporality 😄 
The New Relic platform does not currently support cumulative metric temporalities.